source/refined-github.css

```diff
@@ -186,3 +186,13 @@ div[data-target='readme-toc.content'] div.blob-wrapper img {
 .ActionListItem .ActionListItem-label--truncate {
 	white-space: unset !important;
 }
+
+html:not([rgh-no-navigation-highlight])
```

Comment:
Unrelated feature or it wouldn't belong to `refined-github.css`; `no-navigation-highlight` _removes_ the hover.
```suggestion
```